mobile/styles/Css.tsx

```diff
@@ -0,0 +1,50 @@
+import { StyleSheet, Dimensions } from 'react-native';
+import { vw, vh, vmin, vmax } from 'react-native-expo-viewport-units';
+// import { useFonts, Poppins_500Medium } from '@expo-google-fonts/poppins';
+
+// var [fonts] = useFonts({
+//     Poppins_500Medium,
+//   });
```

Comment:
Can these comments be deleted? if it's temporary or test code you can use stash or a test branch and commit WIP locally